charts/pulsar/templates/bookkeeper-cluster-initialize.yaml

```
@@ -16,7 +16,7 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-{{- if or .Release.IsInstall .Values.initialize }}
+{{- if or (and .Values.useReleaseStatus (or .Release.IsInstall .Values.initialize)) .Values.initialize }}
```

It seem that the condition could be simplified 
```suggestion
{{- if or (and .Values.useReleaseStatus .Release.IsInstall) .Values.initialize }}
```

---

Thanks for the feedback, update pushed.